python/requirements/ml/requirements_jax.txt

```diff
@@ -0,0 +1,8 @@
+tensorflow==2.9.1; python_version >= '3.7'
```

Comment:
Suggestion: Add a comment here that these files are only being used for tests right now. (Since it's not being used for docker images)